metacatalog/ext/standard_export/util.py

```diff
@@ -0,0 +1,627 @@
+
+                # append to bbox_location
+                bbox_locations.append({'min_lon': min_lon, 'min_lat': min_lat, 'max_lon': max_lon, 'max_lat': max_lat})
+
+    # raise ValueError if location is neither specified in datasource.spatial_scale nor in Entry.location
+    if not bbox_locations:
+        raise ValueError("No location information associated with instance to be exported.")
+
+    return temporal_scales, bbox_locations, spatial_resolutions
+
+
+def _parse_iso_information(entry_or_resultset: Union[Entry, ImmutableResultSet]) -> dict:
+    """
+    Loads the ImmutableResultSet of the input Entry (if not already an ImmutableResultSet) 
+    and extracts the information necessary for ISO export.
+
+    Parameters
+    ----------
+    entry_or_resultset : Union[Entry, ImmutableResultSet]
+        The entry instance to be exported
+    
+    Returns
+    ----------
+    iso_input: dict
+        Dictionary of information 
+
+    """
+    if not isinstance(entry_or_resultset, ImmutableResultSet):
+        # get ImmutableResultSet
+        rs = ImmutableResultSet(entry_or_resultset)
+    else:
+        rs = entry_or_resultset
+
+    # uuid / fileIdentifier
+    uuid = _get_uuid(rs)
+
+    # lastUpdate, convert to date, convert to isoformat
+    lastUpdate = _get_lastUpdate(rs)
+
+    # title
+    title = _get_title(rs)
+
+    # publication, convert to date, convert to isoformat
+    publication = _get_publication(rs)
+
+    # version
+    version = _get_version(rs)
+
+    # authors (last_name, first_name, organisation_name, role), always as list of dicts
+    authors = _get_authors(rs)
+
+    # abstract
+    abstract = _get_abstract(rs)
+
+    # details_table, put details into field <abstract> as markdown table for now
+    details = _get_details(rs)
+
+    # keywords (full_path, thesaurusName.title)
+    keywords = _get_keywords(rs)
+
+    ### licenses (link, short_title)
+    licenses = _get_licenses(rs)
+
+    ### datasource (spatial_scale.resolution, spatial_scale.extent/bbox_location, temporal_scale.extent, temporal_scale.resolution)
+    temporal_scales, bbox_locations, spatial_resolutions = _get_datasource_information(rs)
+
+    # save everything in dict
+    iso_input = {
+        'uuid': uuid, 'lastUpdate': lastUpdate, 'publication': publication, 'version': version, 'title': title, 
+        'authors': authors, 'abstract': abstract, 'details': details, 'keywords': keywords, 'licenses': licenses,
+        'temporal_scales': temporal_scales, 'bbox_locations': bbox_locations, 'spatials_resolutions': spatial_resolutions
+        }
+
+    return iso_input
+
+
+def _validate_xml(xml: str) -> bool:
+    """
+    Checks whether the input XML is well-formed (correct syntax).
+    Currently, it is not checked whether the input XML is also valid (We could e.g.
+    check with schematron for the 20 mandatory fields).
+    """
+    try: 
+        etree.fromstring(xml)
+    except etree.XMLSyntaxError as e:
+        raise(e)
```

Comment:
please add an empty line at the end of the file.

Also, this function is not returning anything...